config.json.example

```diff
@@ -1,3 +1,19 @@
 {
-    "token": "000000000:AAAAAAaAAa2AaAAaoAAAA-a_aaAAaAaaaAA"
+    "token": "000000000:AAAAAAaAAa2AaAAaoAAAA-a_aaAAaAaaaAA",
+    "ircBlacklist": [
+        "CowSayBot"
+    ],
+    "irc": {
+        "server": "irc.freenode.net",
+        "channel": "#channel",
+        "botName": "teleirc"
+    },
+    "tg": {
+        "chatId": "-0000000000000",
+        "spacer": ":",
+        "showJoinMessage": true,
+        "showActionMessage": true,
+        "showLeaveMessage": false,
+        "showKickMessage": false
+    }
 }
```

Comment:
you _COULD_  rename this to .js -- and simply export the JSON object -- by simply requiring it, you get a javascript object.